app/src/main/java/org/wikipedia/dataclient/wikidata/Entities.kt

```diff
@@ -0,0 +1,47 @@
+package org.wikipedia.dataclient.wikidata
+
+import org.wikipedia.dataclient.mwapi.MwResponse
+import org.wikipedia.json.PostProcessingTypeAdapter.PostProcessable
+
+class Entities : MwResponse(), PostProcessable {
+
+    val entities: Map<String, Entity>? = null
+        get() = field ?: emptyMap()
+    val first: Entity?
+        get() = entities?.values?.iterator()?.next()
```

Comment:
If the chagnes are done above, this can be as following:
```
val first: Entity?
        get() = if (entities.isEmpty()) null else entities.values.iterator().next()
```